tests/phpunit/integration/Core/Authentication/Clients/OAuth2Test.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * OAuth2Test.php
+ *
+ * @package   Google\Site_Kit\Tests\Core\Authentication\Clients
+ * @copyright 2022 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Authentication\Clients;
+
+use Google\Site_Kit_Dependencies\GuzzleHttp\Psr7\Utils;
+use Google\Site_Kit_Dependencies\GuzzleHttp\Psr7\Query;
+use Google\Site_Kit_Dependencies\Psr\Http\Message\RequestInterface;
+use Google\Site_Kit\Core\Authentication\Clients\Google_Site_Kit_Client;
+use Google\Site_Kit\Tests\TestCase;
+
+/**
+ * @group Authentication
+ * @group failing
+ */
+class OAuth2Test extends TestCase {
+
+	/**
+	 * Parse request body stream.
+	 *
+	 * @param RequestInterface $request The request to parse body for.
+	 * @return array The parsed body.
+	 */
+	private static function getParsedRequestBody( $request ) {
```

Comment:
Sorry I missed it the first time round, but please can you change this method name to snake case in line with WordPress PHP naming conventions (I know it's a bit of a mix in this PR, with good reason).

```suggestion
	private static function get_parsed_request_body( $request ) {
```

Reply:
Uh oh, sorry for mixing it up. Updated, thank you!